nixos/modules/services/monitoring/prometheus/exporters/pihole.nix

```diff
@@ -16,14 +16,6 @@ in
         pi-hole API token which can be used instead of a password
       '';
     };
-    interval = mkOption {
```

Comment:
Use `lib.mkRemovedOptionModule` to mark the removal.

Reply:
Also this is release note worthy.

Reply:
Fixed both, thanks for letting me know about that, I've learned something new.